modules/ROOT/pages/zuora/zuora-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,152 @@
+|Create revenue schedule for invoice item adjustment (distribute by date range)
+
+|Remove an event trigger
+|DELETE Event Trigger
+
+|Delete usage crud
+|Delete usage record
+|===
+
+=== Changes in Fields and Values
+
+There are some changes in fields and their corresponding values:
+
+* The credit memo items operation and the taxation items created in the `write-off` operation are now applied to the corresponding invoice items and invoice taxation items respectively.
+* The values of the following fields are also changed to the taxation items of credit memo items operation:
+  ** taxRate +
+  The value of this field is now set to `0`.
+  ** exemptAmount +
+  The value of this field is now set to `0`.
+  ** taxRateType + 
+  The value of this field is now set to `null`.
+
+
+== Limitations
+
+AQuA operations are removed from Zuora Connector. To use AQuA operations, install Zuora AQuA Connector.  
+
+After you upgrade to Zuora Connector 5.0.0, the following operations are no longer available in Zuora Connector and must be replaced with the corresponding Zuora AQuA Connector operations:
+
+|===
+|AQuA operations in Zuora 4.2.x |Zuora AQuA 1.0.0 operations
+
+|Aqua get job results
+|get job results
+
+|Aqua get-last-job-completed
+|get-last-job-completed
+
+|Aqua post-query-or-export-deleted-data
+|post-query-or-export-deleted-data
+
+|Aqua delete-a-running-job
+|delete-a-running-job
+|===
+
+
+== Upgrade Prerequisites
+
+Before you perform the upgrade, you must:
+
+. Create a backup of your files, data, configuration in case you need to restore to the previous version. 
+. Install Zuora AQuA Connector to replace the AQuA operations that were previously included in Zuora Connector 4.2.x:
+.. In Studio, create a Mule project.
+.. In the Mule Palette, click *Search in Exchange*.
+.. In Add Modules to Project, type 'Zuora AQuA' in the search field.
+.. In Available modules, select *Zuora AQuA Connector*, and then click *Add*.
+.. Click *Finish*.	
+.. Verify that the `zuora-aqua-connector` dependency version is `1.0.0` in the pom.xml. +
+Anypoint Studio upgrades the connector automatically.
+
+== Upgrade Steps
+
+Follow these steps to perform the upgrade to Zuora Connector 5.0.0:
+
+.. In Studio, create a Mule project.
+.. In the Mule Palette, click *Search in Exchange*.
+.. In Add Modules to Project, type 'Zuora AQuA' in the search field.
+.. In Available modules, select *Zuora AQuA Connector*, and then click *Add*.
+.. Click *Finish*.
+.. Verify that the `zuora-connector` dependency version is `5.0.0` in the pom.xml. 
+
+Anypoint Studio upgrades the connector automatically. 
+
+
+== Verify the Upgrade
+
+After you install the latest version of the connector, follow these steps to verify the upgrade:
+
+. In Anypoint Studio, verify there are no errors in the *Problems* or *Console* views.
+. Check the project pom.xml and verify there are no problems.
+. Test the connection and verify the operations work.  
```

Comment:
```suggestion
. Test the connection and verify that the operations work.  
```

Reply:
RE: use of "verify" vs. "verify that":

When you use the verb "verify," it is normally followed by a noun or a "that"-clause. 
A that-clause would normally contain a verb, like "solve" in the following example:

We need to verify the new version. 
We need to verify that the new version of the library solves the following issues list.